app/(main)/admin/components/SongsTable.tsx

```
@@ -0,0 +1,191 @@
+            header: 'Release Date',
+            accessor: (song: Song) => song.releasedate || 'N/A'
+          }
+        ]}
+        page={page}
+        onPageChange={setPage}
+        rowActions={(song: Song) => (
+          <div className="flex gap-2">
+            <button
+              onClick={(e) => { e.stopPropagation(); handleRowClick(song); }}
+              className="text-[--md-sys-color-primary]"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)